### DISCUSSION TITLE
how to change web UI url from 127.0.0.1:8088 to 0.0.0.0:8088

### DISCUSSION THREAD

**Discussion by ZhuJohnson** [microsoft/nni#3749] [Q&A] [answered]
**Environment**:
- NNI version:1.9
- NNI mode (local|remote|pai): in docker of remote server
- Client OS: Linux version 4.13.0
- Server OS (for remote mode only):
- Python version:3,6,8
- PyTorch/TensorFlow version:tensorflow 2.3
- Is conda/virtualenv/venv used?:
- Is running in Docker?:Yes

**Log message**:
 - nnimanager.log: 
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**What issue meet, what's expected?**:
I need the change the web UI url to 0.0.0.0:8088 from 127.0.0.1:8088 since only the port 0.0.0.0:8088 is forwarded from my container to the server.
**How to reproduce it?**: 

**Additional information**:
Want to have more option to specify host except port number when we run the experiement, and actually this is doable in similar application/packages: such as when I use mlflow ui to manage/track modeling process, I can use mlflow ui --host=0.0.0.0 to specify host address of web UI

**Comment by liuzhe-lz** [comment, marked as answer]
It seems impossible. `0.0.0.0` is a special address and cannot be used as destination IP. Normally it either stands for "any IP" or "no IP".
